src/components/saveSelector.html

```diff
@@ -0,0 +1,20 @@
+<div class="d-flex align-items-center justify-content-center background-night py-4" id="saveSelector">
+    <div class="justify-content-center container row">
+        <div class="col-6">
+            <img style="width:100%" src="assets/images/pokeclickerlogo.png">
+        </div>
+        <div class="col-12 d-flex align-items-center justify-content-center my-3">
+            <a href="https://discord.gg/a6DFe4p">
+                <img alt="Join our Discord!" src="https://img.shields.io/discord/450412847017754644?color=7289DA&label=Discord&logo=discord&logoColor=7289DA&style=for-the-badge">
+            </a>
+            &nbsp;
+            <a href="https://github.com/pokeclicker/pokeclicker">
+                <img alt="Visit our GitHub!" src="https://img.shields.io/github/package-json/v/pokeclicker/pokeclicker/develop?label=GitHub&logo=github&style=for-the-badge">
+            </a>
```

Comment:
Is this pointing to the current version, or the develop version?

Reply:
It's pointing to the dev version, live version would probably cause less confusion though I guess 😅